src/main/java/org/spongepowered/api/command/CommandMessageFormatting.java

```diff
@@ -32,12 +32,18 @@
     private CommandMessageFormatting() {
     }
 
+    public static final Text FORWARD_SLASH = Text.of("/");
```

Comment:
Is there any way to have any additional formatters here? By plugins?

Catalog candidate?